viadot/flows/mindful_to_adls.py

```diff
@@ -33,14 +37,17 @@ def adls_bulk_upload(
         List[str]: List of paths
     """
 
-    for file in file_names:
-        file_path = str(adls_file_path + "/" + file)
-        file_to_adls_task.run(
-            from_path=os.path.join(file_name_relative_path, file),
-            to_path=file_path,
-            sp_credentials_secret=adls_sp_credentials_secret,
-            overwrite=adls_overwrite,
-        )
+    if not file_names:
+        logger.warning("Avoided uploading any file to ADLS. No files were reported.")
```

Comment:
It's the same structure, but the intentions are completely different.
In the task class, I only do that to return `None` and to not let flow fails (that is what the client wants). But I put that in the Flow-task, because it is supposed that that task is going to become a general task in viadot. I think more convenient the warning there instead of in the task class.